src/main/java/core/basesyntax/service/operations/OperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public interface OperationHandler {
+    Map<String, Integer> storageFruits = Storage.fruits;
```

Comment:
According to the encapsulation principle, you should not expose the internal data of the `Storage` class. Instead of directly accessing the `storageFruits` map from the interface, consider using methods provided by the `Storage` class to interact with the storage.